sdk/aqueduct/artifacts/bool_artifact.py

```diff
@@ -74,19 +74,19 @@ def get(self, parameters: Optional[Dict[str, Any]] = None) -> Union[bool, np.boo
                     "Parameterizing historical artifacts is not currently supported."
                 )
 
-        if parameters is None and self._get_content() is not None:
-            return self._get_content()
+        content = self._get_content()
+        if parameters is not None or content is None:
+            previewed_artifact = artifact_utils.preview_artifact(
+                self._dag, self._artifact_id, parameters
+            )
+            content = previewed_artifact._get_content()
+            assert isinstance(content, bool) or isinstance(content, np.bool_)
```

Comment:
is this necessary?

Reply:
should line 89 also cover this?

Reply:
yeah it isn't (see answer below), removing.